rts/lra/lra-filters/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -395,6 +401,30 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
         }
     }
 
+    private void checkForRecovery(ContainerRequestContext requestContext) {
+        // if the request is expecting a recovery check
+        if (requestContext.getUriInfo().getQueryParameters().containsKey(LRAConstants.RECOVERY_PARAM) &&
```

Comment:
sorry for possibly silly question - but where is `LRAConstants.RECOVERY_PARAM` set on the `requestContext` please?

Reply:
In the MP-LRA TCK replayEndPhase call

Reply:
Here is a [link](https://github.com/eclipse/microprofile-lra/blob/478f51ff93ab90e726b6ada3269952d4a77e0d45/tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTestBase.java#L166)